src/sites/link/1ink.cc.js

```diff
@@ -0,0 +1,13 @@
+_.register({
+  rule: {
+    host: /^1ink\.cc$/,
+    path: /^\/\w+$/,
+  },
+  async ready () {
+    let url = document.head.querySelector('[name=keywords]').content;
+    const urlCheck = url.includes('http');
+    if (urlCheck != true)
```

Comment:
Just `!urlCheck` if it's just a boolean. Also, do not omit braces for the `if` statement.

Reply:
Already changed it.
Thank you for the advices.